app/src/main/java/com/nervousfish/nervousfish/activities/ContactActivity.java

```diff
@@ -23,6 +23,7 @@
 /**
  * The ContactActivity shows the contacts information and his public keys.
  */
+@SuppressWarnings("checkstyle:missingctor")
```

Comment:
If you're going to suppress this so often, why not ignore it in the configuration?